<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTilemap</class>
 <widget class="QWidget" name="DlgTilemap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tilemap</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tilemap image file</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QToolButton" name="btnSelectImage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource/resource.qrc">
            <normaloff>:/16x16_ico_png/folder.png</normaloff>:/16x16_ico_png/folder.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="imageFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Select tilemap image file (none selected)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Tile region position in tilemap (top left corner)</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Left is at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="offsetX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Top is at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="offsetY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Tile dimensions</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Width</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tileWidth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
             <property name="value">
              <number>32</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tileHeight">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
             <property name="value">
              <number>32</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Padding</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="padding">
             <property name="suffix">
              <string> px</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="gui::GfxWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="zoom">
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbColorSpace"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Min filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbMinFilter"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Mag filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbMagFilter"/>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPremulAlpha">
          <property name="text">
           <string>Premultiply alpha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPremulAlphaBlend">
          <property name="text">
           <string>Blend with premultiplied alpha</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkGrid">
          <property name="text">
           <string>Show guide grid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="color_widgets::ColorSelector" name="widgetColor">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorSelector</class>
   <extends>color_widgets::ColorPreview</extends>
   <header>color_selector.hpp</header>
  </customwidget>
  <customwidget>
   <class>gui::GfxWidget</class>
   <extends>QWidget</extends>
   <header>gfxwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>imageFile</tabstop>
  <tabstop>btnSelectImage</tabstop>
  <tabstop>offsetX</tabstop>
  <tabstop>offsetY</tabstop>
  <tabstop>tileWidth</tabstop>
  <tabstop>tileHeight</tabstop>
  <tabstop>padding</tabstop>
  <tabstop>widget</tabstop>
  <tabstop>zoom</tabstop>
  <tabstop>cmbColorSpace</tabstop>
  <tabstop>cmbMinFilter</tabstop>
  <tabstop>cmbMagFilter</tabstop>
  <tabstop>chkPremulAlpha</tabstop>
  <tabstop>chkPremulAlphaBlend</tabstop>
  <tabstop>chkGrid</tabstop>
  <tabstop>widgetColor</tabstop>
  <tabstop>btnExport</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
